src/olympia/editors/models.py

```diff
@@ -78,7 +78,7 @@ def new_editors():
 
         return [dict(user=i.arguments[1],
                      created=i.created)
-                for i in items]
+                for i in items if i.arguments[1] in group.users.all()]
```

Comment:
The issue is not super important, it just annoyed me. I thin I'd be fine with the current solution, we can revisit it later...